### DISCUSSION TITLE
Announcing intention to change release cadence for the project

### DISCUSSION THREAD

**Discussion by vojtapolasek** [ComplianceAsCode/content#11278] [Announcements]
Hello all,
on behalf of contributors who are regularly publishing releases of this project, I would like to announce intention to change cadence of those releases.
Currently, we publish new release every two months. 
We would like to prolong the period between two releases to three months.
The stabilization phase stays the same; two weeks.
This should bring two main benefits. Firstly, there are certain security policies (for example DISA STIG) which are updated roughly every three months. Currently, it happens that the time of policy update and the time of the project release desynchronize over time, meaning that the release might not contain the content aligned with the latest published policy and at the same time content consumers have to wait for more than month for the updated content. We are aware that this can happen due to many circumstances, but changing the release cadence should eliminate the most obvious one.
Secondly, this will save the release team some time which is spent on reviewing of stabilization jobs every two months. We believe that we could use this time to improve health of the project as well as release automation without impacting usefulness of releases.
The date of upcoming release 0.1.71 stays the same.
Now is the time for your feedback. Feel free to voice your opinions and questions here.
If we do not receive any negative feedback until Monday November 27th, we plan to start doing releases every three months, counting from the 0.1.72 release.
Thank you and have a nice day.

**Comment by vojtapolasek** [comment]
@ComplianceAsCode/ubuntu-maintainers  @ComplianceAsCode/oracle-maintainers  @ComplianceAsCode/suse-maintainers  @ComplianceAsCode/red-hatters

**Comment by marcusburghardt** [comment]
I see benefits on this and really like the idea of more automation in the process. I agree.

**Comment by dodys** [comment]
Sounds good to me :)

**Comment by vojtapolasek** [comment]
As there was no negative feedback, we will be switching to three month cadence of upstream releases. The 0.1.72  will happen in February, and further releases will happen in three month intervals.
